scripts/bootstrap-aio.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+#
+# Copyright 2014, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## Shell Opts ----------------------------------------------------------------
+set -e -u -x
+
+## Variables -----------------------------------------------------------------
+# Extra options to pass to the AIO bootstrap process
+export BOOTSTRAP_OPTS=${BOOTSTRAP_OPTS:-''}
+
+export BASE_DIR=${BASE_DIR:-"/opt/rpc-openstack"}
+export OA_DIR="${BASE_DIR}/openstack-ansible"
+export RPCD_DIR="${BASE_DIR}/rpcd"
```

Comment:
@alextricity25 I would recommend against that. Right now they will work if one of the scripts, or the user overrides them but the scripts will operate on their own too. If you're trying to be DRY I guess it could be done, but I don't really see much value from it.